src/main/java/eu/europa/ted/eforms/sdk/component/SdkComponentFactory.java

```diff
@@ -126,22 +142,40 @@ protected <T> T getComponentImpl(String sdkVersion, final SdkComponentType compo
   }
 
   protected <T> T getComponentImpl(String sdkVersion, final SdkComponentType componentType,
-      final String qualifier, final Class<T> intf, Object... initArgs) throws InstantiationException {
+      final String qualifier, final Class<T> intf, Object... initArgs)
+      throws InstantiationException {
 
     String normalizedVersion = normalizeVersion(sdkVersion);
 
     ComponentSelector selector = new ComponentSelector(componentType, qualifier);
 
+    Map<ComponentSelector, SdkComponentDescriptor<?>> map =
+        Optional.ofNullable(componentsMap.get(normalizedVersion))
+            .orElseGet(Collections::emptyMap);
+
+    if (logger.isDebugEnabled()) {
```

Comment:
This should rather be done at the "trace" level.

Reply:
OK i will use trace

Reply:
This has not been changed, it is still done at "debug" level".